src/styles/index.scss

```diff
@@ -1,3 +1,16 @@
+html {
+  font-family: Roboto, serif;
+  font-weight: 400;
+}
+
 body {
   margin: 0;
 }
+
+p {
+  margin: 0;
+}
+
+@import './variables';
+@import './card';
```

Comment:
The `card.scss` file is imported here. Make sure it is correctly compiled into CSS along with this `index.scss` file.